test/integration/storage/Storage.test.js

```diff
@@ -170,31 +170,32 @@ describe('Storage', () => {
 
         expect(results).toEqual([msg1, msg2, msg3, msg4, msg5])
     })
-
+    /* Doesn't seem like a relevant test anymore since messages cannot be created without msgChainId
```

Comment:
Hmm.. indeed the test in its current form is not relevant. However, we should test the case "fetch messages starting from a timestamp, sequenceNo for a given publisher" (i.e. `msgChainId`  removed). Is this tested elsewhere, if not, this test could be modified to test this case?